src/App.vue

```diff
@@ -978,6 +998,31 @@ export default class App extends Mixins(
     await updateLdUser(key, email, firstName, lastName, custom)
   }
 
+  /** Stores business info from Legal API. */
+  storeBusinessInfo (response: any): void {
+    const business = response?.data?.business as BusinessIF
+
+    if (!business) {
+      throw new Error('Invalid business info')
+    }
+
+    // if (this.getBusinessId !== business.identifier) {
+    //   throw new Error('Business identifier mismatch')
+    // }
+
+    // FUTURE: change this to a single setter/object?
+    this.setAdminFreeze(business.adminFreeze)
+    this.setEntityName(business.legalName)
+    this.setEntityState(business.state)
```

Comment:
can we use one value to set all these business objects? 
So we can have one mutation for business object

Reply:
I looked into that and it may be difficult/messy since there are other places in the code that set just one of these values.

Maybe we could "set" an array of key-val tuples, and the action would only set the keys (properties) in the list?

Reply:
I think we can leave it like this , may more easy to read code